kano/utils/file_operations.py

```diff
@@ -201,6 +201,31 @@ def write_json(filepath, data, prettyprint=False, sort_keys=True):
             run_cmd(cmd)
 
 
+def touch(path, times=None):
+    """Set the access and modified times of the file specified by path.
+
+    This is essentially a simple wrapper for :func:`os.utime`.
+
+    Args:
+        path (str): Path to the file create/modify
+        times (tuple): See :func:`os.utime`
+
+    Returns:
+        bool: Whether the operation was successful or not
+    """
+    try:
+        with open(path, 'a'):
+            os.utime(path, times)
+    except (IOError, OSError) as error:
```

Comment:
Would you like to try and create a file in a dir as the path if it's not there? Or is it just about timestamping? 

```
(...)
except OSError:
        os.makedirs(os.path.dirname(path))
        open(path, "w").close()
        os.utime(path, times)
```

Reply:
That's right or pass `dir` as a parameter to the wrapper:

```
def touch(path, times=None, dirs=False)        
    """
    A nice description.

    Args:
        path: file path to touch
        times: tuple blah blah defaults to current time.
        dirs: is set, create folders if not exists 
    """
    
    if dirs:
        basedir = os.path.dirname(path)
        if not os.path.exists(basedir):
            os.makedirs(basedir)
            
    with open(path, 'a'):
        os.utime(path, times) 
```

Reply:
Called the function beforehand to handle that automatically 👍